modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/FunctionsTest.java

```diff
@@ -100,6 +100,9 @@ public void testRangeWithCache() throws Exception {
 
         awaitPartitionMapExchange();
 
+        GridTestUtils.assertThrowsWithCause(() -> checkQuery("SELECT t._key, (SELECT x FROM TABLE(system_range(1, 5))) FROM \"test\".Integer t").check(),
```

Comment:
I would prefer not to extend a random test with an additional check.

It's better to extend `org.apache.ignite.internal.processors.query.calcite.exec.rel.BaseAggregateTest` with a few additional tests to verify singe_value accumulator behaviour.

For integration tests there is `org.apache.ignite.internal.processors.query.calcite.AggregatesIntegrationTest`.